x-pack/plugins/security_solution/public/management/pages/policy/view/artifacts/layout/policy_artifacts_layout.test.tsx

```diff
@@ -185,5 +196,47 @@ describe('Policy artifacts layout', () => {
       await render(false);
       expect(renderResult.queryByTestId('artifacts-assign-button')).toBeNull();
     });
+    it('should not display assign artifacts button on empty state when there are artifacts', async () => {
+      mockedApi.responseProvider.eventFiltersList.mockImplementation(
+        (args?: { query: { filter: string } }) => {
+          if (
+            !args ||
+            args.query.filter !== parsePoliciesAndFilterToKql({ policies: [policyItem.id, 'all'] })
+          ) {
+            return getFoundExceptionListItemSchemaMock(1);
+          } else {
+            return getFoundExceptionListItemSchemaMock(0);
+          }
+        }
+      );
+      await render(false);
+      expect(await renderResult.findByTestId('policy-artifacts-empty-unassigned')).not.toBeNull();
+      expect(renderResult.queryByTestId('unassigned-assign-artifacts-button')).toBeNull();
+    });
+    it('should not display manage artifacts button on empty state when there are artifacts', async () => {
+      mockedApi.responseProvider.eventFiltersList.mockImplementation(
+        (args?: { query: { filter: string } }) => {
+          if (
+            !args ||
+            args.query.filter !== parsePoliciesAndFilterToKql({ policies: [policyItem.id, 'all'] })
+          ) {
+            return getFoundExceptionListItemSchemaMock(1);
+          } else {
+            return getFoundExceptionListItemSchemaMock(0);
+          }
+        }
+      );
+      await render(false);
+      expect(await renderResult.findByTestId('policy-artifacts-empty-unassigned')).not.toBeNull();
+      expect(renderResult.queryByTestId('unassigned-manage-artifacts-button')).toBeNull();
+    });
```

Comment:
You could combine these two tests.

Reply:
https://github.com/elastic/kibana/pull/147676/commits/59f4cda79e1f18cfe4277ca734172b334ebd98a8